apps/web/src/studio/LocalStudioAuthenticator.tsx

```diff
@@ -110,9 +105,14 @@ export function LocalStudioAuthenticator() {
     const localBridgeURL = buildBridgeURL(parsedApplicationOrigin.origin, tunnelPath);
     const tunnelBridgeURL = buildBridgeURL(tunnelOrigin, tunnelPath);
 
+    // TODO: Add apiKeys to the IEnvironment interface as they exist in the response
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-expect-error
+    const devSecretKey = environments.find((env) => env.name.toLowerCase() === 'development')?.apiKey;
```

Comment:
@SokratisVidros just to clarify, is the apiKey here is defined on the environments object?  I remember it was an array of `apiKeys[0].key`

Reply:
It does on the API layer, but the Interface in @novu/shared is not up to date. However this change will happen in my PR that removes envId from the JWT.